src/components/shared/Entity/Error/index.tsx

```diff
@@ -30,10 +30,13 @@ function EntityError({ logToken, error, title, draftId }: Props) {
         <HeaderSummary severity="error" title={title}>
             <Stack direction="column" spacing={2}>
                 <Box>
-                    <Error error={error} hideTitle={true} />
+                    <Error error={error} hideTitle={true} noAlertBox />
 
                     {idForDraftErrors ? (
-                        <DraftErrors draftId={idForDraftErrors} />
+                        <>
+                            <Divider />
+                            <DraftErrors draftId={idForDraftErrors} />
+                        </>
```

Comment:
Throwing in a divider for design purpose. Now that the above does not have a box having these errors indented looked a bit better after a line on the page.